ceph-docker-nightly/build/build

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
 
 # the following two methods exist in scripts/build_utils.sh
-pkgs=( "pytest" )
+pkgs=( "pytest" "docker-py==1.10.3" )
```

Comment:
I think I know why you're doing this (API in 2 is different?) and it's not related to this commit, right?